base/unicode.jl

```diff
@@ -1,10 +1,8 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 include("unicode/UnicodeError.jl")
-include("unicode/types.jl")
-include("unicode/checkstring.jl")
+# include("unicode/checkstring.jl")
 include("unicode/utf8.jl")
-include("unicode/utf16.jl")
 include("unicode/utf32.jl")
```

Comment:
That file isn't actually deleted here yet – there are a bunch of method definitions in it that weren't actually specific to the UTF32String type. For now I'm focusing on deleting things and keeping the tests working rather than moving things around as well. Once everything is pared down and working, then reorganization can be done.